lib/ukargparse/tests/test_ukargparse_parse.c

```diff
@@ -0,0 +1,96 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Simple memory pool using LIFO principle
+ *
+ * Authors: Rico Muench <muench.rico@gmail.com>
+ *
+ *
+ * Copyright (c) 2020, NEC Laboratories Europe GmbH, NEC Corporation,
+ *                     All rights reserved.
```

Comment:
You can contribute the copyright to yourself, it does not necessarily need to be NEC :)